src/pages/order/CheckBox.js

```diff
@@ -0,0 +1,89 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const CheckBox = () => {
+  const [checkedList, setCheckedLists] = useState([]);
+
+  const onCheckedAll = checked => {
+    if (checked) {
+      const checkedListArr = [];
+      CHECKBOX_AGREEMENT.forEach(Agreementlist =>
+        checkedListArr.push(Agreementlist)
+      );
+      setCheckedLists(checkedListArr);
+    } else {
+      setCheckedLists([]);
+    }
```

Comment:
```suggestion
   const newCheckedList = checked ? [...CHECKBOX_AGREEMENT] : [];
   setCheckedList(newCheckedList);
```

불필요하게 명령형으로 써져있는 것 같네요 선언적으로 간단하게 표현할 수 있어보입니다!